### ISSUE TITLE
[AcmeClient] Call to undefined function system_webgui_configure()

### ISSUE THREAD

**Issue by smindorf** [opnsense/plugins#236]
Hi, 
after upgrade to OPNsense 17.7-amd64 the acme-client will not work correctly.

**Error**:
```
root@fw01:/usr/local/opnsense/scripts # /usr/local/opnsense/scripts/OPNsense/AcmeClient/certhelper.php -F -a sign -A

Fatal error: Uncaught Error: Call to undefined function system_webgui_configure() in /usr/local/opnsense/scripts/OPNsense/AcmeClient/certhelper.php:1019
Stack trace:
#0 /usr/local/opnsense/scripts/OPNsense/AcmeClient/certhelper.php(273): run_restart_actions(Array, Object(OPNsense\AcmeClient\AcmeClient))
#1 /usr/local/opnsense/scripts/OPNsense/AcmeClient/certhelper.php(85): cert_action_validator(NULL)
#2 {main}
  thrown in /usr/local/opnsense/scripts/OPNsense/AcmeClient/certhelper.php on line 1019

```

I checked the code and found the systemcall: `system_webgui_configure()` but I can't find
the function on my system.
I've checked `/usr/local/etc/inc/*.inc` not here.

**Instelled Plugins:**

- os-acme-client	1.8
- os-dyndns	1.1
- os-haproxy	1.16
- os-intrusion-detection-content-pt-open 1.0
- os-rfc2136	1.0
- os-smart	1.1

Regards
Sören

**Comment by fichtner** [comment]
It's true, the function no longer exists since a 17.1.x and was not really supposed to be public.

**Comment by fichtner** [comment]
A configd call existed, but was removed because unused. Now we have our use case ;)

**Comment by fraenki** [comment]
@fichtner Is there a configd command available to reload the WebUI?

**Comment by fichtner** [comment]
see second commend, I'll add it back in a minute or so

**Comment by fichtner** [comment]
@fraenki see https://github.com/opnsense/core/commit/bc261c67b8610c861783a80de7233620372d61ee -- shall I commit a fix for the plugin?

**Comment by fraenki** [comment]
@fichtner OK :)

**Comment by fichtner** [comment]
@fraenki I've taken the safe road with a delay of 2 seconds in the background, but maybe you can get away with a simple synchronous non-delay reload. This is something you can try as you have the setups.

@smindorf Thank you for the report. It will be fixed in 17.7.1 :)
